src/main/scala/uk/gov/nationalarchives/PuidRepository.scala

```diff
@@ -0,0 +1,48 @@
+package uk.gov.nationalarchives
+
+import cats.effect.kernel.Async
+import cats.implicits._
+import doobie.Transactor
+import doobie.implicits._
+import doobie.util.transactor.Transactor.Aux
+import uk.gov.nationalarchives.DatabaseConfig.DatabaseCredentials
+import uk.gov.nationalarchives.PuidRepository.{AllPuidInformation, PuidInformation}
+
+class PuidRepository[F[_]: Async](xa: Aux[F, Unit]) {
+
+  private def allowedPuids(): F[List[PuidInformation]] = {
+    val query = sql"""SELECT "PUID", 'NonJudgmentFormat'::text FROM "AllowedPuids"; """.query[PuidInformation].to[List]
```

Comment:
The `AllowedPuids` table contains the puids that are valid for a judgment, so not sure this is correct this `NonJudgmentFormat`?

Reply:
The 'NonJudgmentFormat' value wasn't even being used, I just did it this way so I could crowbar both this and the disallowedPuids results into the same case class. 

That doesn't really make sense so I've got rid of the `'NonJudgmentFormat'::text` and created a new Case class for `AllowedPuids` which just contains the PUID.